python/lambda.py

```diff
@@ -6,6 +6,9 @@
 
 from across_api.base.api import app
 from across_api.across.api import *  # noqa F401
+from across_api.burstcube.api import *  # noqa F401
+from across_api.swift.api import *  # noqa F401
```

Comment:
I think that we just need to run the FastAPI function decorators; we don't actually need to pull these into the module namespace, do we?

```suggestion
import across_api.across.api  # noqa F401
import across_api.burstcube.api  # noqa F401
import across_api.swift.api  # noqa F401
```